tailwind.config.js

```diff
@@ -1,9 +1,9 @@
 /** @type {import('tailwindcss').Config} */
-export default {
+module.exports = {
```

Comment:
Why changing from `export default` to `module.exports` ?

Reply:
Cancel that change please. It'll lead to issues

Reply:
Sir, the change has already been made in the last commit